save-backend/src/main/kotlin/org/cqfn/save/backend/configs/WebSecurityConfig.kt

```diff
@@ -61,6 +74,26 @@ class WebSecurityConfig(
         .logout().disable()
         .formLogin().disable()
         .build()
+
+    @Bean
+    fun projectPermissionEvaluator() = ProjectPermissionEvaluator()
+
+    fun roleHierarchy(): RoleHierarchy = mapOf(
+        Role.ADMIN to listOf(Role.PROJECT_OWNER, Role.PROJECT_ADMIN, Role.VIEWER),
```

Comment:
let's remove OWNER and have ADMIN and VIEWER only (HOPE we won't need it)

Reply:
Removed